model-optimizer/mo/utils/cli_parser.py

```diff
@@ -1032,6 +1032,17 @@ def get_mean_scale_dictionary(mean_values, scale_values, argv_input: str):
     if type(mean_values) is dict and type(scale_values) is dict:
         if not mean_values and not scale_values:
             return res
+
+        for inp_scale in scale_values.keys():
+            if inp_scale not in inputs:
```

Comment:
Is there a possibility that we haven't set input, but set scale without a name because we have only one input in the network?

Reply:
Interestingly but the MO still produces correct result in this case (the Multiply op appears in the IR)...

But anyway, let's check that there is a unit test covering this case.

Reply:
thanks for the feedback. I haven't considered this case. I didn't pay attention to comment on line 1119 
# mean and scale are specified without inputs, return list, order is not guaranteed (?)
I'll check this case.

Reply:
I double checked this scenario. Everything works properly and unittests are already written.